code/modules/mob/living/taste.dm

```diff
@@ -15,10 +15,34 @@
 		. = 101 // can't taste anything without a tongue
 
 // non destructively tastes a reagent container
+
+//FermiChem - How to check pH of a beaker without a meter/pH paper.
+//Basically checks the pH of the holder and burns your poor tongue if it's too acidic!
+//TRAIT_AGEUSIA players can't taste, unless it's burning them.
+//taking sips of a strongly acidic/alkaline substance will burn your tongue.
 /mob/living/proc/taste(datum/reagents/from)
+	var/obj/item/organ/tongue/T = getorganslot("tongue")
```

Comment:
```suggestion
	var/obj/item/organ/tongue/T = getorganslot(ORGAN_SLOT_TONGUE)
```
Use defines here and on similar procs such as `get_bodypart()`, from now on.

Reply:
Done